terms.html

```diff
@@ -81,8 +81,8 @@
   <dt><dfn class="lint-ignore"
            data-lt="identities|identity's">identity</dfn></dt>
   <dd>
-The means for keeping track of <a>entities</a> across contexts. Digital
-identities enable tracking and customization of <a>entity</a> interactions
+The means for keeping track of <a>entities</a> and other <a>resources</a> across contexts.
```

Comment:
'Entities' are defined as "A thing with distinct and independent existence, such as a person, organization, or device that performs one or more roles in the ecosystem." It's what can perform the roles of holder, issuer and verifier.

'Resource' (as I added it from the [RDF primer](https://www.w3.org/TR/rdf11-primer/#introduction)) is "Anything, including documents, people, physical objects, and abstract concepts, that information can be expressed about".

So yes, an entity is a resource, but the converse is not true: a diamond tiara's is obviously a resources, but isn't an entity, as it  cannot perform a role in the ecosystem.

The utility of this is that saying that only entities can perform the roles of holder, issuer and verifier implies that if you're not an entity, you can't.

Reply:
-1 to adding resource... it doesn't add enough value. The cost for adding more terms is cognitive load on people that needs to understand the basic ecosystem at first glance... The place to get into these sorts of details is later in the specification, not in the definition.

Reply:
I think it is very valuable (if not crucial) to express that claims can be about other things than the entities that issue, hold or verify them. So we need a term for whatever claims can be made about. The place where we list such terms with their definitions is the terminology section. Further explanations can go elsewhere. Also, people that need to understand the basic ecosystem at first glance do not get that understanding from the terminology section. That's what an introduction, or an overview section would be more appropriate for.

Reply:
I might be misunderstanding the various conversations here, but they seem to revolve around the idea that an entity cannot be, for example, an abstract concept. We should just make clear that it can be in its definition.

An entity can be a person, place, thing, idea, i.e., anything. Each issuer, verifier, and holder is an entity. I don't yet understand why this approach is insufficient.

That an entity can be an abstract concept does not mean that an abstract concept can be an issuer. I think people will happily assume that an entity that is also an issuer necessarily has the capabilities required to be able to issue VCs -- something an abstract concept cannot do. None of this seems like a problem to me.

I feel we're overly complicating our definitions and should try to make the above adjustment to keep it simpler.

Reply:
The conversations are about (a) distinguishing between two concepts, and (b) their naming.

The first concept indicates who/what can 'perform one or more roles in the ecosystem', i.e. be an issuer, holder, or verifier. There are various options for this, e.g.: 
- individuals or organizations, that decide e.g., about semantics, how well the truth of assertions is  established, and that can be held accountable,
- devices, that can, e.g., assert measurements in claims, yet won't be accountable,
- both (and perhaps there are other choices)
Indeed, there have been discussions around this that have led to choose for the last option (both), and call that 'entities' - see its current definition.

The second concept indicates who/what is that assertions can be made about. While it is obvious that assertions can be made about 'entities' as currently defined, there are other things - tangible as well as intangible (e.g.: abstract concepts) - that assertions can be made about. As we've seen various discussions expressing confusion about what can, or cannot be this 'who/what that assertions can be made about', this concept, too merits clarification. In my opinion the description of what the RDF primer calls a `resource` is a great start, and that's why I proposed it. 

Note that we should clearly distinguish between (1) the concepts we want to define and (2) the names we want to use to refer to such concepts. We should include a description (definition) of the concepts we need within the VCDM in its terminology section, and then choose a name that minimizes confusion. That's converse to what is typically done, in statements such as `<term> doesn't mean what's defined!`. So I'm fine with any changes in the names we associate with a concept (which is different from changing the description of the concept).

Reply:
I have reluctantly removed 'resource', in an attempt to get this PR done.